packages/spanner/src/database.js

```diff
@@ -787,6 +787,8 @@ Database.prototype.runStream = function(query, options) {
  * atomically at a single logical point in time across columns, rows, and tables
  * in a database.
  *
+ * Note that Cloud Spanner does not support nested transactions.
```

Comment:
I think we should elaborate on what exactly that means in the context of this client. For eg something like.
```
Note that Cloud Spanner does not support nested transactions. If a new transaction is started inside the "run function" it will be an independent transaction."
```

Also it might make sense to then put this comment after the paragraph below since that is where we define what is a "run fucntion"